circuits-circom/regexes/venmo_actor_id.circom

```diff
@@ -189,4 +189,21 @@ template VenmoActorId (msg_bytes) {
     for (var i = 0; i < num_bytes; i++) {
         reveal[i] <== in[i] * states[i+1][1];
     }
+
+    signal output reveal_start_idx;
+    signal prefix_sum[num_bytes+1];
+    signal temp_index[num_bytes+1];
+    component is_eq_one[num_bytes];
+    prefix_sum[0] <== 0;
+    temp_index[0] <== 0;
+    for (var i = 1; i <= num_bytes; i++) {
+        prefix_sum[i] <== prefix_sum[i-1] + (1 - states[i][1]);
```

Comment:
Should `(1 - states[i][1])` be `(1 * states[i][1])` instead? That way prefix_sum is only 1 at the first nonzero states value.

Reply:
fixed here https://github.com/zkp2p/zk-p2p/pull/119/commits/f0062fce89cf44fd63aa33c12856bf96c286f942